### ISSUE TITLE
We should have an iOS version 🙂

### ISSUE THREAD

**Issue by EdwardHinkle** [cleverdevil/Indiepaper-macOS#2]
👍

(Originally published at: https://eddiehinkle.com/)

**Comment by EdwardHinkle** [comment]
The iOS version has been submitted to TestFlight. Once we have a Privacy Policy (https://github.com/cleverdevil/indiepaper-website/issues/4), it'll be submitted to the iOS App Store.

**Comment by cleverdevil** [comment]
Happy to put a privacy policy in place. Do you have one for Indigenous that we could replicate? If not, I’ll spend some time researching tomorrow and put forth a best effort policy based upon what I find.

**Comment by EdwardHinkle** [comment]
Yeah, here’s Indigenous’ privacy policy (https://indigenous.abode.pub/ios/privacy/). I based it off of Overcast’s privacy policy. So you can take anything from Indigenous or Overcast as you see fit.
